FAQ/security.md

```diff
@@ -27,6 +27,10 @@ If you don't recognize the key, are they tricking verifiers into thinking you ar
 
 Expired private keys should be deleted, the corresponding public keys should stay in the issuer published key set to allow verifiers to validate health cards issued using them. Revoked private keys (compromised, issued in error, etc.) should be deleted and removed from the published key set.
```

Comment:
Can we add an optional `exp` field to the key object in the `jwks.json` to allow for a key that is known to have been compromised after a certain date?  In that case, only SHCs signed with that key after the `exp` value would be automatically invalidated.

Reply:
Putting an expiration date on a key has been discussed, and could be useful on its own merit, but I'd tackle that in a separate thread (PR). I do think that limiting the time range of a key for revocation purposes would be useful. True @baptistelaget, a key leak would allow back signing, but an unauthorized key access (e.g., by a rogue employee having access to an issuance system for a limited time period, without access to the private key itself) could be identified up to a certain datetime, and the adverse effects of the revocation could be limited by keeping a subset of the issued SHCs valid. This being said, I'd rather push this PR as is, and debate the need for this feature separately; short key lifetimes might be enough to address this use case without adding complexity to the revocation system.

Reply:
Good points, @baptistelaget and @christianpaquin.  Agree with pushing the PR as-is with respect to this.